hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/pbj/PbjConverter.java

```diff
@@ -668,6 +678,14 @@ public static com.hederahashgraph.api.proto.java.TopicID fromPbj(TopicID id) {
                 .build();
     }
 
+    public static com.hederahashgraph.api.proto.java.TokenID fromPbj(TokenID id) {
```

Comment:
@NonNull +  Nullcheck for id and @NonNull for method result

Reply:
Done